cpp/src/io/json/write_json.cu

```diff
@@ -545,8 +542,10 @@ struct column_to_strings_fn {
       column.null_count(),
       cudf::detail::copy_bitmask(column, stream_, rmm::mr::get_current_device_resource()),
       stream_);
-    auto l_pre = string_scalar{"["}, l_post = string_scalar{"]"}, sep = string_scalar{","},
-         elem_narep = string_scalar{"null"};
+    auto l_pre      = string_scalar{"[", true, stream_};
+    auto l_post     = string_scalar{"]", true, stream_};
+    auto sep        = string_scalar{",", true, stream_};
+    auto elem_narep = string_scalar{"null", true, stream_};
```

Comment:
This could be moved and added as class member.

Reply:
You mean use the `narep` class member here instead of `elem_narep`?

Reply:
No. I meant, all 4 scalars could be moved to the class. Right now, this scalar is initialized everytime on a list column call.

Reply:
Since we already have `include_nulls` in options to control null behaviour for structs and table, `narep` could be kept common for both struct and list. List will always include nulls, so list_narep cannot be null.

Reply:
Ok. I made them share a `narep` scalar value. Let me know if that looks right.